src/geometry.h

```diff
@@ -0,0 +1,1104 @@
+#pragma once
+
+#include "turner.h"
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <cmath>
+#include <string>
+
+namespace turner {
```

Comment:
Should this be `namespace turner { namespace geometry {} }` or am I confusing packages and namespaces?